### PR TITLE
Cover page

### PR DESCRIPTION
**Pull request by andrewjtimmons**
# what?

a pr to add a coverpage to a bulk voter download. 

Because the pdf creation for each individual pdf is async the order of the cover page may not match the order of the pdfs.

This can be fixed in the future by either
A) making pdf generation syncronous (slower, but more likely to succeed with large lists since it wont queue up 100 pdf jobs at once)
B) keeping track of the order things finish and then recording the voter list or the order of pdf files.  

### notes on callback heck and await/syncronous heck
http://callbackhell.com/
https://medium.freecodecamp.org/avoiding-the-async-await-hell-c77a0fb71c4c

# testing
tested on dev 

# risk
relatively minor, just adding a template

### CODE REVIEW

**Comment by andrewjtimmons** [comment]
also i tried using async await but node complained.  probably have to upgrade something.  imma take a nap instead tho and blow my nose :D

**Comment by sjforman** [comment]
Ok, so, yes, this looks great and definitely works. I pushed a small change, adding the full address and adjusting some styling. I also see now what you mean about the callback confusion — it is...kind of hard to follow. Probably worth refactoring at some point. But, definitely serves the purpose for now. Merging. Thank you!
